src/ui/handler/drag_pan.js

```diff
@@ -147,28 +154,39 @@ class DragPanHandler {
 
         this._state = 'pending';
         this._startPos = this._mouseDownPos = this._prevPos = this._lastPos = DOM.mousePos(this._el, e);
+        this._startTouch = this._lastTouch = e instanceof window.TouchEvent ? DOM.touchPos(this._el, e) : null;
         this._inertia = [[browser.now(), this._startPos]];
     }
 
+    _touchesMatch(lastTouch: ?Array<Point>, thisTouch: ?Array<Point>) {
+        if (!lastTouch || !thisTouch || lastTouch.length !== thisTouch.length) return false;
+        for (let i = 0; i < lastTouch.length; i++) {
+            if (!lastTouch[i].equals(thisTouch[i])) return false;
+        }
+        return true;
```

Comment:
small thing, but you could use `.every` to clean it up a bit.

Reply:
Fixed in https://github.com/mapbox/mapbox-gl-js/pull/8100/commits/b50f108fa85748c6c139a3e2fb4f237305b2a9fe